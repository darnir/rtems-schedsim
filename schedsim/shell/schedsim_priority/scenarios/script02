echo "*** TEST 02 ***"
rtems_init
echo "=== Create 0x0a010001 ==="
task_create TA1 1
task_wake_after 5
clock_tick 7
echo "*** END OF TEST 2 ***"
exit
# We will not get here
