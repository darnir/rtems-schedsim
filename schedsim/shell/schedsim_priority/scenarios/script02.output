*** TEST 02 ***
  Thread Heir: 0x09010001 priority=255
  Thread Executing: 0x09010001 priority=255
=== Create 0x0a010001 ===
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010001, priority=1
Task (TA1) starting: id=0x0a010001, priority=1
  Thread Heir: 0x0a010001 priority=1
  Thread Executing: 0x0a010001 priority=1
Task (0x0a010001) sleeping for 5 ticks
  Thread Heir: 0x09010001 priority=255
  Thread Executing: 0x09010001 priority=255
ClockTick (1) ...
ClockTick (2) ...
ClockTick (3) ...
ClockTick (4) ...
ClockTick (5) ...
  Thread Heir: 0x0a010001 priority=1
  Thread Executing: 0x0a010001 priority=1
ClockTick (6) ...
ClockTick (7) ...
*** END OF TEST 2 ***
