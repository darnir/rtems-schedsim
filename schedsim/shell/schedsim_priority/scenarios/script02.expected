Script File               : script02.scen
verbose                   : 1
==> 1: echo "*** TEST 02 ***"
*** TEST 02 ***
==> 2: rtems_init
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
==> 3: echo "=== Create 0x0a010001 ==="
=== Create 0x0a010001 ===
==> 4: task_create TA1 1
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010001, priority=1
Task (TA1) starting: id=0x0a010001, priority=1
  CPU 0: Thread Heir: 0x0a010001 priority=1
  CPU 0: Thread Executing: 0x0a010001 priority=1
==> 5: task_wake_after 5
Task (0x0a010001) sleeping for 5 ticks
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
==> 6: clock_tick 7
ClockTick (1) ...
ClockTick (2) ...
ClockTick (3) ...
ClockTick (4) ...
ClockTick (5) ...
  CPU 0: Thread Heir: 0x0a010001 priority=1
  CPU 0: Thread Executing: 0x0a010001 priority=1
ClockTick (6) ...
ClockTick (7) ...
==> 7: echo "*** END OF TEST 2 ***"
*** END OF TEST 2 ***
==> 8: exit
